docs/essentials/exceptions/exceptions-create-user-defined.md

```diff
@@ -1,6 +1,6 @@
 # How to create user-defined exceptions
```

Comment:
Yeah, I think this could fall under "Creating and Raising Exceptions".  There isn't a whole lot to creating user-defined exceptions, so it could get folded into that other article.